common/src/main/java/com/wynntils/wynn/model/map/MapModel.java

```diff
@@ -109,4 +175,13 @@ public static CompletableFuture<Boolean> tryLoadMaps() {
     public static boolean isMapLoaded() {
         return !maps.isEmpty();
     }
+
+    private static class PlacesProfile {
+        List<Label> labels;
+    }
+
+    private static class ServiceProfile {
```

Comment:
Do I understand correctly that these "profiles" are basically deserialization helpers?

Reply:
Yes. We've traditionally used `*Profile` as name for stuff whose structure are forced upon us by external data. It's kind of odd, but also marks the classes clearly. But in this case I could perhaps rename them to something like `ServiceProfileJsonDeserializer` to make this more explicit.